README.md

```diff
@@ -39,6 +40,27 @@ Unlike `vde_vmnet`, `socket_vmnet` does not depend on VDE.
 
 Requires macOS 10.15 or later.
 
+### From binary
```

Comment:
From github release?

Reply:
I'm not sure. The source releases (git tags and source archives) are also "GitHub release" IIUC.